db_upgrade_script/mosip_esignet/sql/1.5.1_to_1.6.0_upgrade.sql

```diff
@@ -0,0 +1,8 @@
+\c mosip_esignet
+
+DO $$
+BEGIN
+
+ALTER TABLE client_detail ADD COLUMN additional_config jsonb;
+
+END $$
```

Comment:
It is just an alter statement it is not required to execute it inside do block @sacrana0